infra/keycloak/themes/alexandria/login/resources/css/styles.css

```
@@ -0,0 +1,235 @@
+/* Alexandria login theme, layered on keycloak.v2 */
+
+:root {
+  --alx-primary: #0b6394;
+  --alx-primary-dark: #084b6f;
+  --alx-ink: #0a2942;
+  --alx-ink-deep: #061c30;
+  --alx-text: #14181d;
+  --alx-text-muted: #545c66;
+  --alx-border: #cbd2da;
+  --alx-radius: 8px;
+  --alx-font: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
```

Stylelint (17.14.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)